src/applications/mhv/medications/util/constants.js

```diff
@@ -72,11 +72,18 @@ export const medicationsUrls = {
   MEDICATIONS_LOGIN: '/my-health/medications?next=loginModal&oauth=true',
 };
 
-export const refillStatus = [
+export const refillStatusForRefillsReft = [
   'active',
   'activeParked',
-  'hold',
   'refillinprocess',
 ];
 
+export const dispStatusForRefillsReft = [
```

Comment:
for the const name what is the 'Reft' part supposed to represent?

Reply:
sorry.  it means 'Left'